e2e/terraform/provision-nomad/main.tf

```diff
@@ -34,6 +34,7 @@ resource "null_resource" "provision_nomad" {
   # Run the provisioner as a local-exec'd ssh command as a workaround for
   # Windows remote-exec zero-byte scripts bug:
   # https://github.com/hashicorp/terraform/issues/25634
+  # https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md#0150-unreleased
```

Comment:
Left myself this note because I keep having to go look up the changelog link again every time I check in on https://github.com/hashicorp/terraform/issues/25634